wp-dependency-installer.php

```diff
@@ -425,7 +425,7 @@ public function install( $slug ) {
 			}
 
 			wp_cache_flush();
-			if ( $this->is_required( $this->config[ $slug ] ) ) {
+			if ( $this->is_required( $slug ) ) {
```

Comment:
Was this an error before?

Reply:
No, it's just clearer to read ( those calls are equivalent: [wiki/Helper-Functions](https://github.com/afragen/wp-dependency-installer/wiki/Helper-Functions))